java-squid/src/main/java/org/sonar/java/se/ExplodedGraphWalker.java

```diff
@@ -77,7 +76,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(ExplodedGraphWalker.class);
   private static final Set<String> THIS_SUPER = ImmutableSet.of("this", "super");
 
-  private static final boolean DEBUG_MODE_ACTIVATED = false;
+  private boolean DEBUG_MODE_ACTIVATED = false;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "DEBUG_MODE_ACTIVATED" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS00116)